demos/coreMQTT/mqtt_demo_mutual_auth.c

```diff
@@ -505,11 +483,7 @@ int RunCoreMqttMutualAuthDemo( bool awsIotMqttMode,
      */
     ulGlobalEntryTimeMs = prvGetTimeMs();
 
-    /* Seed randon number generator for use with backoff algorithm library for backoff calculation
-     * during retry attempts of failed network operations with the broker. */
-    xRngSeedStatus = prvSeedRandomNumberGenerator();
-
-    for( ; ( ( xRngSeedStatus == pdPASS ) && ( ulDemoRunCount < democonfigMQTT_MAX_DEMO_COUNT ) ); ulDemoRunCount++ )
+    for( ; ( ulDemoRunCount < democonfigMQTT_MAX_DEMO_COUNT ); ulDemoRunCount++ )
```

Comment:
This usage is a little out of the ordinary, so the reader is forced to scan backwards to find the last assignment of `ulDemoRunCount`.  Make it easier for them by just doing the normal thing here:

```c
for( ulDemoRunCount = 0 ; ...
```

Reply:
Updated for improved user readability